Combinatorial_map/doc/Combinatorial_map/Concepts/GenericMapItems.h

```diff
@@ -35,6 +36,19 @@ The following examples show two possible models of the `GenericMapItems` concept
 */
 class GenericMapItems {
 public:
+  /*!
+    \cgalModifBegin
+    If Use_index is equal to CGAL::Tag_true, use indices as descriptors. Otherwise (if this type is not defined or different than CGAL::Tag_true), use handles.
```

Comment:
```suggestion
    If `Use_index` is  `CGAL::Tag_true`, uses indices as descriptors. Otherwise (if this type is not defined or different from `CGAL::Tag_true`), use handles.
```